src/hydrate/values/readEmailIfExists.test.ts

```diff
@@ -0,0 +1,79 @@
+import { describe, expect, it, vi } from "vitest";
+
+import { readEmailIfExists } from "./readEmailIfExists.js";
+
+const validAuthorPackage = {
+	author: "Author<author@package.com>",
+	email: "author@package.com",
+};
+
+const validEmailPackage = {
+	author: { email: "info@package.com", name: "Author" },
+};
+
+const mock$ = vi.fn();
+
+vi.mock("execa", () => ({
+	get $() {
+		return mock$;
+	},
+}));
+
+const mockGetNpmUserInfo = vi.fn();
+
+vi.mock("../../shared/getNpmUserInfo.js", () => ({
+	get getNpmUserInfo() {
+		return mockGetNpmUserInfo;
+	},
+}));
+
+describe("readEmailIfExists", () => {
+	it('reads email from the package "author" field', async () => {
```

Comment:
[Nitpick] This is the nittiest of the nitpicks, but I normally write test names in _"it X when Y"_ format to make it clear what both X and Y are at a glance. This is super nitpicky and if you hate it no worries, I can make the change myself 😄 

```suggestion
	it('reads email from the package "author" field when it exists', async () => {
```